viewer/src/utilities/networking/CogniteClient3dExtensions.ts

```diff
@@ -3,7 +3,6 @@
  */
 
 import { CogniteClient, CogniteInternalId, CogniteExternalId, IdEither } from '@cognite/sdk';
-import { HttpHeaders } from '@cognite/sdk/dist/src/utils/http/basicHttpClient';
```

Comment:
Why was this import removed?

Reply:
Because it's internal to @cognite/sdk, ref `dist + we don't need it.